website/views/shared/profiles/achievements.jade

```diff
@@ -146,8 +146,8 @@ div(ng-if=':: profile.achievements.originalUser')
   
 div(ng-if='profile.achievements.challenges || user._id == profile._id')
   // This is a very strange icon to use. revisit
-  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges')
-  div(ng-class='{muted: !profile.achievements.challenges}')
+  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges.length')
```

Comment:
`ng-show` and `ng-hide` control the `display` property of the element with classes, so they're still present on the dom. `ng-if` removes it from the dom entirely if it doesn't satisfy the condition, which technically improves the performance of the web-app, though the performance boost would probably be so small, it'd be negligible. There are some tricky considerations to make when using `ng-if` over `ng-show`, but they don't apply to this circumstance. 

Here's a stackoverflow page that goes into some more depth: http://stackoverflow.com/questions/19177732/what-is-the-difference-between-ng-if-and-ng-show-ng-hide